astroquery/exoplanets/planet_params.py

```diff
@@ -0,0 +1,153 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import astropy.units as u
+from astropy.time import Time
+from astropy.coordinates import SkyCoord
+
+__all__ = ['PlanetParams']
+
+
+class PlanetParams(object):
+    """
```

Comment:
I struggle to understand the raison d'être of the `PlanetParams` class.  The only benefit I can see of calling `PlanetParams.from_exoplanet_archive("Trappist-1 b").pl_radj` instead of `ExoplanetArchive.query_planet("trappist-1 b")['pl_radj']` is that the former returns a Quantity and the latter a float, but this could be resolved by using a `QTable` instead of a `Table` in the implementation of `ExoplanetArchive`.

I don't see `StarParams` or `GalaxyParams` classes in astroquery, so I wonder why there should be a `PlanetParams`?  I would consider this carefully because adding this class means more code to maintain whilst risking confusing the user by providing two ways to do the same.

Reply:
I've come around, I see it your way now. `PlanetParams` only offers cosmetic differences in the API, and it will probably more trouble than it's worth to maintain. I'm looking into returning a QTable now.